packages/skygear-core/lib/container.js

```diff
@@ -673,77 +242,121 @@ export default class Container {
     return this._store;
   }
 
-  get publicDB() {
-    if (this._publicDB === null) {
-      this._publicDB = new Database('_public', this);
-      this._publicDB.cacheResponse = this._cacheResponse;
-    }
-    return this._publicDB;
-  }
-
-  get privateDB() {
-    if (this.accessToken === null) {
-      throw new Error('You must login before access to privateDB');
-    }
-    if (this._privateDB === null) {
-      this._privateDB = new Database('_private', this);
-      this._privateDB.cacheResponse = this._cacheResponse;
-    }
-    return this._privateDB;
+  clearCache() {
+    return this.store.clearPurgeableItems();
   }
 
   get Database() {
     return Database;
   }
 
+}
+
+export default class Container extends BaseContainer {
+
+  constructor() {
+    super();
+
+    this._auth = new AuthContainer(this);
+    this._relation = new RelationContainer(this);
+    this._db = new DatabaseContainer(this);
+    this._pubsub = new PubsubContainer(this);
+    this._push = new PushContainer(this);
+    /**
+     * Options for how much time to wait for client request to complete.
+     *
+     * @type {Object}
+     * @property {number} [timeoutOptions.deadline] - deadline for the request
+     * and response to complete (in milliseconds)
+     * @property {number} [timeoutOptions.response=60000] - maximum time to
+     * wait for an response (in milliseconds)
+     *
+     * @see http://visionmedia.github.io/superagent/#timeouts
+     */
+    this.timeoutOptions = {
+      response: 60000
+    };
+  }
+
```

Comment:
Please add esdocs annotation to `Container`. At least to make the doc navigtable, and we can fill in the detail laters.